query_insert.go

```diff
@@ -83,6 +83,16 @@ func (q *InsertQuery) Column(columns ...string) *InsertQuery {
 	return q
 }
 
+func (q *InsertQuery) ColumnExpr(query string, args ...interface{}) *InsertQuery {
+	q.addColumn(schema.SafeQuery(query, args))
+	return q
+}
```

Comment:
Do you have an example of the query with ColumnExpr? If so we may add such query to https://github.com/uptrace/bun/blob/master/internal/dbtest/query_test.go#L396-L403 so we don't break it.

Reply:
@vmihailenco I'd assume anything that requires calling a stored procedure, a static value or a built in function would benefit from ColumnExpr, eg: ColumnExpr("NOW()") or ColumnExpr("SomeStoredProcedureWithReturn()")

Reply:
I believe it should be `Value("column", "now()")` - see https://github.com/uptrace/bun/pull/22. I am going to remove `ColumnExpr` since it does not produce valid query.

Reply:
Makes sense, can be removed I guess